cit3.0-web/src/components/Page/citHome/citHome.js

```diff
@@ -1,136 +1,231 @@
+                <p className="my-2">
+                  Community and regional profiles include socioeconomic data,
+                  infrastructure and community assets to provide a sense of what
+                  a community is like – and how it is changing.{" "}
+                </p>
+                <>
+                  {places ? (
+                    <div className="w-100 mt-4 comm-search-div">
+                      <Typeahead
+                        id="search-by-community"
+                        size="large"
+                        onChange={(selected) => setSelectedPlace(selected[0])}
+                        options={places}
+                      />
+                      <Button
+                        className="search-glass-box"
+                        variant="outline-secondary"
+                      >
+                        <BsSearch />
+                      </Button>
+                    </div>
+                  ) : null}
+                </>
+              </Col>
+            </Row>
+            <Row className="pb-0 d-flex justify-content-end">
+              <Col sm={3} className="svg-box pt-3">
+                <img
+                  className="add-opp-img"
+                  src="/images/house_CITHOME.svg"
+                  height="100%"
+                  width="100%"
+                  alt="cit home community search box"
+                />
+              </Col>
+            </Row>
+          </Col>
+          <Col className="box ml-3 cit-box explore-box corner">
+            <Row>
+              <Col>
+                <h3>Discover insights and patterns among B.C. communities</h3>
+                <p className="my-2">
+                  Find all communities in B.C. with particular characteristics –
+                  whether you’re interested in economic health, access to
+                  education and health care, connectivity, infrastructure, or
+                  emergency management.
+                </p>
+              </Col>
+            </Row>
+            <Row>
+              <Col className="mt-2 d-flex justify-content-end">
+                <SharedButton
+                  onClick={handleExploreClick}
+                  styling="home-buttons explore-button"
+                  label="Explore B.C. Communities"
+                />
+              </Col>
+            </Row>
+            <Row className="mb-0 pr-0 mt-3 d-flex justify-content-end">
+              <Col sm={6} className="p-0 mb-0 svg-box">
+                <img
+                  className="add-opp-img"
+                  src="/images/HouseMountain.svg"
+                  height="100%"
+                  width="100%"
+                  alt="cit home community search box"
+                />
+              </Col>
+            </Row>
+          </Col>
+        </Row>
+      </Container>
+
+      <Modal
+        show={show}
+        onHide={handleClose}
+        backdrop="static"
+        keyboard={false}
+      >
+        <Modal.Header>
```

Comment:
Getting rid of the `backdrop="static"` and adding a closeButton prop to the header allows us to close the modal by clicking a button or by clicking the backdrop, might be something worth adding?
```suggestion
      <Modal
        show={show}
        onHide={handleClose}
        keyboard={false}
      >
        <Modal.Header closeButton>
```